README.md

```diff
@@ -18,7 +18,7 @@ that will suggest people matching an entered text.
 - Install Prettier Extention and use this [VSCode settings](https://mate-academy.github.io/fe-program/tools/vscode/settings.json) to enable format on save.
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_autocomplete/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<igols>.github.io/react_autocomplete/) and add it to the PR description.
```

Comment:
The demo link in PR description is wrong, you need to fix here and update it in the PR description. Also you need add all this link, not only [DEMO LINK]
<img width="1003" alt="image" src="https://github.com/user-attachments/assets/18244b28-2f6a-44e9-b2f5-8e4d49c02ef2" /> 
```suggestion
- Replace `<your_account>` with your Github username in the [DEMO LINK](https://igols.github.io/react_autocomplete/) and add it to the PR description.
```